analyzere/base_resources.py

```diff
@@ -328,7 +328,7 @@ def upload_data(self, file_or_str, chunk_size=analyzere.upload_chunk_size,
             request_raw('patch', self._data_path, headers=headers, body=chunk)
             # if there is a known size, and an upload callback, call it
             if length:
-                upload_callback((offset/length) * 100.0)
+                upload_callback((offset / length) * 100.0)
```

Comment:
this might be a little weird due to python2/3 compatibility.  I think in python2 it's integer division, which may still be totally fine because it only has to be approximate and never greater than 1.

Reply:
Great catch!

It might not belong in this PR, but I'm going to fix it here.